modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -17,23 +17,24 @@ permissions.
 
 . Navigate in the web console to the *Catalog → Operator Hub* page.
 
-. Click *Show Community Operators*.
+. Choose the Operator you want to install from the list of available Operators (*etcd* for this example), and click *Install*. A pop-up window appears, warning you that you are installing a Community Operator.
 +
 [NOTE]
 ====
 This Operator will be available under the *Red Hat* category in the GA release.
 ====
+. Select *Continue* to acknowledge the Show Community Operator warning. The Install page appears.
 
-. Choose the Operator you want to install from the list of available Operators (*etcd* for this example), and click *Install*.
+. Read the information about the Operator and click *Install*.
 
-. On the *Create Operator Subscription* page, select *All namespaces...* 
-under Installation Mode, choose an Update Channel (if more than one is available),
-And select *Automatic* or *Manual* Approval Strategy, as described earlier.
+. On the *Create Operator Subscription* page, select:
+.. *A specific namespace on the cluster* under Installation Mode to choose a namespace in which the Operator will be available (*default* installs to all namespaces)
```

Comment:
This doesn't seem right to me: "(*default* installs to all namespaces)"

This sounds like if you select "A specific namespace on the cluster" and choose "default" from the drop down, that it installs to all namespaces. But I don't think that's the case - at least when I test it, I only see the etcd options showing in Developer Catalog when I choose the "default" project, no others.

Also seems odd because the "All namespaces on the cluster" option is disabled.

Reply:
That is weird. Maybe something like this: Select *All namespaces on the cluster (default)* to install the Operator to all namespaces or *A specific namespace on the cluster* to choose a specific, single namespace on which to install the Operator. The "All namespaces" option is not always available.